### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by Goose-Of-War**
* Form base template and styling added
* Auth frontend and backend implemented

### CODE REVIEW

File: test/site-auth.js
Comment (ItsAnkan) [inline test/site-auth.js:1]: Shows this error
```
C:\Users\ANKAN\Desktop\Code n' Stuff\MASK-Event\node_modules\axios\dist\node\axios.cjs:836
  AxiosError.call(axiosError, error.message, code, config, request, response);
```

Comment (ItsAnkan) [reply test/site-auth.js:1]: and this `AxiosError: connect ECONNREFUSED ::1:7359`

Comment (Goose-Of-War) [reply test/site-auth.js:1]: In the very last post request, right?

Comment (ItsAnkan) [reply test/site-auth.js:1]: yea

Comment (SachdevJai) [reply test/site-auth.js:1]: where :question:

Comment (ItsAnkan) [reply test/site-auth.js:1]: > where ❓

run `npm test` and check its after the last test

Comment (Goose-Of-War) [reply test/site-auth.js:1]: It happens after passing only. Works ig

Comment (ItsAnkan) [reply test/site-auth.js:1]: so we ignor ?

Comment (Goose-Of-War) [reply test/site-auth.js:1]: Safe to ignore at the moment ig

**Review by ItsAnkan — APPROVED** [review]
Looks fine to me. May need some Button css improvements, but all in all, its okay.

**Pull request merged**